setup-sshkeys.md

```diff
@@ -0,0 +1,94 @@
+# Setting up SSH Public/Private Keys
+
+We recommend setting up SSH public/private keys to access UNIX/Linux
+systems quickly and effectively:
+
+1. [What is a public-private key] (#what-is-a-public-private-key)
+2. [How to create a public-private key] (#how-to-create-a-public-private-key)
+3. [Keeping remote systems up to date] (#keeping-remote-systems-up-to-date)
+
+-----
+
+#### What is a Public-Private key
+
+SSH (on UNIX/Linux systems) supports login via public/private key.
+The idea here is that you never get prompted for username or password
+again when logging on to UNIX/Linux systems. It works from putty on
+Windows as well, along with SecureCRT and other popular terminal
+emulators supporting the SSH protocol. Very, very useful.
+
+Also, [PBUILD](https://github.com/MSFTOSSMgmt/pbuild) requires use of
+public/private keys to work properly. This mechanism only works if you
+use a non-shared account. I DO NOT recommend that you set this up on
+some shared account (i.e. root).
+
+
+#### How to Create a Public-Private Key
+
+To create a public/private key:
+
+* Use ssh-keygen. Do this from your “primary” UNIX machine (where you do
+most of your development). This will be something like:
+```
+ssh-keygen -t dsa -b 2048 -C "Microsoft-internal"
+```
+
+Resulting output will be something like:
+
+>
+```
+Generating public/private dsa key pair.
+Enter file in which to save the key (/home/jeffcof/.ssh/id_dsa): 
+Enter passphrase (empty for no passphrase): 
+Enter same passphrase again: 
+Your identification has been saved in /home/jeffcof/.ssh/id_dsa.
+Your public key has been saved in /home/jeffcof/.ssh/id_dsa.pub.
+The key fingerprint is:
+68:8e:50:3c:8d:cc:fa:77:2b:ad:cc:88:d6:5f:2f:4a Microsoft-internal
+```
+>
+
+* In general, we don't recommend the use of a passphraise, although
+that is up to you. Most UNIX/Linux systems are shared systems with
+sudo access and shared root passwords (i.e. inherently insecure). If
```

Comment:
The "shared root password" doesn't look very serious from an outsider's point of view. Maybe remove it.